odd-collector/odd_collector/adapters/postgresql/models.py

```diff
@@ -181,6 +181,15 @@ def dependencies(self) -> list[Dependency]:
             return []
 
 
+@dataclass(frozen=True)
+class Relationship:
+    fk_constraint_name: str
+    source_table: str
+    source_fk_column_name: str
+    target_table: str
+    target_fk_column_name: str
```

Comment:
We have a quite strict context here - posgresql. So I would stick postgresql terminology here.
For example:
```
CREATE TABLE sportspeople(
    sportsperson_id INT,
    sport_id INT,
    sportperson_name VARCHAR(255) NOT NULL,
  PRIMARY KEY(sportsperson_id),
  CONSTRAINT fk_sport_id     // constraint_name
  FOREIGN KEY(sport_id)       //     foreign_key
  REFERENCES sports(sport_id)  // references
  ON DELETE NO ACTION
 );
```
```suggestion
class Relationship:
    constraint_name: str
    source_table: str
    foreign_key: tuple[str]     // we need tuple here because it's can be combination of keys and the order matters
    target_table: str
    references: tuple[str]
```
I believe it's more correct from terminology and semantics due we have postgresql context here.
cc: @RamanDamayeu @Vladysl

Reply:
Looks logic for me. But attribute name `references` in combination with  `foreign_key` seems strange. In your sql example `REFERENCES ...` is a verb, but in `class Relationship` we are waiting for attribute name to be a noun, so it became plural noun `references`. I think both attributes that describe connection should be in one state: if `foreign_key` then `reference`. But in our case we can have more than 1 appearing, so maybe it is worth to stick to `foreign_keys` and `references` then?